jupyterhub/orm.py

```diff
@@ -204,7 +204,7 @@ class Spawner(Base):
     server = relationship(Server, cascade="all")
 
     state = Column(JSONDict)
-    name = Column(Unicode(255))
+    server_name = Column(Unicode(255))
```

Comment:
This is the name of the Spawner, so I think it's redundant to use `spawner.server_name` here. Let's leave the Spawner.name field alone.

Reply:
Your logic makes sense to me -- I appreciate the clarification and feedback!